### PR TITLE
Feature/fallback content

### PR DESCRIPTION
**Pull request by domingobishop**
Please review changes

### CODE REVIEW

File: inc/functions-admin.php [hunk 1]
```diff
@@ -5,6 +5,26 @@
...
 	add_menu_page( 'Homepage settings', 'Homepage', 'administrator', 'homepage-admin-page', 'homepage_admin_page', 'dashicons-admin-generic', 21  );
+
+	add_action( 'admin_init', 'homepage_admin_page_settings' );
+}
+
+function homepage_admin_page_settings() {
+	register_setting( 'homepage-settings-group', 'home_masthead_descr' );
+	register_setting( 'homepage-settings-group', 'home_masthead_btn_text_1' );
+
+	register_setting( 'homepage-settings-group', 'home_landing_page_title_1' );
```
Comment (punalsc) [inline inc/functions-admin.php]: The register setting seems repetitive. I'm happy with the code, but if there are further requirements in the future to add more, i would suggest using a loop, which will keep the code DRY.

File: functions.php [hunk 2]
```diff
@@ -1,61 +1,28 @@
...
-    return is_array($var) ? array_intersect($var, array('current-menu-item')) : '';
-}
-add_filter('nav_menu_css_class', 'attributes_filter', 100, 1);
-add_filter('nav_menu_item_id', 'attributes_filter', 100, 1);
-add_filter('page_css_class', 'attributes_filter', 100, 1);
-
-// Include functions
-include 'functions-home.php';
-
+add_action( 'wp_enqueue_scripts', 'tna_chile_scripts' );
```
Comment (punalsc) [inline functions.php]: Spelling error
`add_action( 'wp_enqueue_scripts', 'tna_chile_scripts' );`

File: js/admin.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Admin JS.
+ */
+
+jQuery(document).ready(function() {
+    jQuery('.media-metabox-button-js').click(function() {
```
Comment (domingobishop) [inline js/admin.js]: Done

File: js/tna-child-home.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+jQuery(document).ready(function() {
+
+    var $count = jQuery(".col-card").length;
+
+    if ($count < 3) {
+        var elements = document.getElementsByClassName("col-card");
+
+        for (var i = 0; i < elements.length; i++) {
+
+            elements[i].style.display = "none";
```
Comment (domingobishop) [inline js/tna-child-home.js]: done

File: js/tna-child-home.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    }
+
+    if ( $count > 3 && $count < 6 ) {
+        var elements = document.getElementsByClassName("col-card");
+
+        for (var i = 3; i < elements.length; i++) {
+
+            elements[i].style.display = "none";
```
Comment (domingobishop) [inline js/tna-child-home.js]: done

File: inc/functions-admin.php (hunk 1 above)
Comment (domingobishop) [inline inc/functions-admin.php]: In progress

File: functions.php (hunk 2 above)
Comment (domingobishop) [inline functions.php]: done

**Comment by punalsc** [comment]
Happy with the fixes.

**Pull request merged**